pages/treeherder.py

```diff
@@ -78,6 +84,11 @@ def filter_by(self, term):
         el.send_keys(Keys.RETURN)
         self.wait.until(lambda s: self.result_sets)
 
+    def filter_unclassified_jobs(self):
+        self.find_element(*self._unclassified_failure_filter_locator).click()
+        self.wait.until(lambda s: len(self.result_sets) == 10)
+        return self
```

Comment:
Yes, there's no value in returning the page object. If that code is already there then you can feel free to ignore it for this patch and perhaps remove it in a separate one.